### PR TITLE
Add multi-country charts; COUNTRY=rbd

### PR DESCRIPTION
**Pull request by jbperidypathtech**
This fixes issue #785.

How to test the feature:

- Go to Charts tab
- Try to get charts when global configuration is for cross-countries (ex. rbd)
- Try to get charts when global configuration is for mono-countries (ex. mozambique)

Screenshot/video of feature:

Screenshot 2023-06-08 at 10 30 10 AM

### CODE REVIEW

**Comment by ericboucher** [comment]
@wadhwamatic for RBD and other multi-country deployments, would you like to leave the country blank, or select the first one in the list by default?

File: frontend/src/config/rbd/prism.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "country": "Regional Bureau Dakar",
+  "multiCountries": true,
```
Comment (ericboucher) [inline frontend/src/config/rbd/prism.json:3]: I think `multiCountry` is the proper English spelling actually

**Comment by ericboucher** [comment]
@jbperidypathtech great work. This seems to be working well.
Now I think we can revamp/simplify the logic in `MapView/LeftPanel/ChartsPanel/index.tsx` and DRY it.
And add a few comments where they can help

File: frontend/src/config/rbd/layers.json [hunk 2]
```diff
@@ -3178,14 +3181,28 @@
...
+      "2019-10-01",
+      "2020-02-01",
+      "2020-06-01",
+      "2020-10-01",
+      "2021-02-01",
+      "2021-06-01",
+      "2021-10-01",
+      "2022-02-01",
+      "2022-06-01",
+      "2022-10-01"
```
Comment (JulienPathtech) [inline frontend/src/config/rbd/layers.json:3199]: Is it normal that there is so much changes in the rdb layer ? For instance here we loose the most recent data point "2023-06-01"

Comment (ericboucher) [reply frontend/src/config/rbd/layers.json:3199]: Nice catch! no it's not normal. I must have pushed some conflicting changes from a different local branch. I'll revert those

**Review by ericboucher — APPROVED** [review]
@jbperidypathtech this is working great, we can merge once you've addressed the remaining small items:
- multiCountry
- merge conflicts

**Comment by wadhwamatic** [comment]
> @wadhwamatic for RBD and other multi-country deployments, would you like to leave the country blank, or select the first one in the list by default?

@ericboucher - let's leave it blank by default

**Review by ericboucher — APPROVED** [review]

**Review by wadhwamatic — APPROVED** [review]
looks good from my side, thanks @jbperidypathtech and @ericboucher

**Pull request merged**